src/deserialize/parse.rs

```diff
@@ -1,7 +1,7 @@
 use crate::edn::{Edn, Error, List, Map, Set, Vector};
 use std::collections::{BTreeMap, BTreeSet};
 
-const DELIMITERS: [char; 5] = [',', ']', '}', ')', ';'];
+const DELIMITERS: [char; 8] = [',', ']', '}', ')', ';', '(', '[', '{'];
```

Comment:
might be missing set delimiter, but we have a workaround to deal with sets, by replacing the `#{` with `@`